src/App.tsx

```diff
@@ -1,50 +1,107 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import './App.scss';
+import cn from 'classnames';
+import { GoodsList } from './components/GoodList/GoodList';
+import { goodsFromServer } from './api/data';
 
-export const goodsFromServer = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
-];
+interface SortTypes {
+  ALPHA: string;
+  LENGTH: string;
+  NONE: string;
+}
+
+const sortFields: SortTypes = {
+  ALPHA: 'alphabet',
+  LENGTH: 'length',
+  NONE: 'string',
+};
```

Comment:
```suggestion
enum SortTypes {
  Alphabet = 'alphabet',
  Length = 'length',
  None = '',
}
```
And use enum instead